tests/integration/targets/lookup_etcd3/aliases

```diff
@@ -4,5 +4,6 @@ needs/file/tests/utils/constraints.txt
 needs/target/setup_etcd3
 skip/aix
 skip/osx
+skip/rhel
```

Comment:
Please do **not** modify unrelated files.

Reply:
Sorry, just testing why tests are failing with etcd.